docs/LanguageSupport.md

```diff
@@ -5,25 +5,33 @@ Below are the Domain Specific Languages (DSL) that Cromwell currently supports a
 ## Current Language Support
 
 ### WDL Draft 2
-Cromwell started life as a WDL engine and that is our recommended execution language. For many examples on how to use WDL and some great getting-started resources you can view [the OpenWDL site](https://github.com/openwdl/wdl#getting-started-with-wdl).
+Cromwell started life as a WDL engine and WDL draft2 was our first language!
```

Comment:
```suggestion
Cromwell started life as a WDL engine and WDL draft 2 was our first language.
```
[ToL] exclamation doesn't feel right to me in factual writing (the other one is OK because it's an inducement to action)